config.go

```diff
@@ -9,8 +9,9 @@ const (
 )
 
 type config struct {
-	StorageType string `envconfig:"STORAGE_ADAPTER" default:"memory"`
-	NumLines    int    `envconfig:"NUMBER_OF_LINES" default:"1000"`
+	StorageType    string `envconfig:"STORAGE_ADAPTER" default:"memory"`
+	NumLines       int    `envconfig:"NUMBER_OF_LINES" default:"1000"`
+	AggregatorType string `envconfig:"AGGREGATOR_TYPE" default:"nsq"`
```

Comment:
can you add in some configs for the channel, topic, host, port, etc...? example is [here](https://github.com/arschles/deis-logger/blob/nsq-lib/config.go#L13-L23)